pkg/madmin/config-commands.go

```diff
@@ -162,3 +163,21 @@ func (adm *AdminClient) SetConfigKeys(params map[string]string) error {
 
 	return nil
 }
+
+// TopLocks - returns the oldest locks in a minio setup.
+func (adm *AdminClient) TopLocks() ([]byte, error) {
```

Comment:
We can have this function in another file other than config-commands.go, it could be locks-commands.go

Reply:
Yes, I will add this function to a new file.